src/install-ocaml-windows.cmd

```diff
@@ -1,5 +1,6 @@
 set CYGWIN_ROOT=c:\cygwin
 %2/setup-x86_64.exe --quiet-mode --root %CYGWIN_ROOT% --site http://cygwin.mirror.constant.com --packages curl,diff,diffutils,git,m4,make,patch,perl,rsync,mingw64-x86_64-gcc-core,unzip
```

Comment:
> > * the command above would be more resilient if it were `%2\setup-x86_64.exe`.
> 
> I'm just using what's already in the CMD file. I'm reluctant to use a separate path, maybe that should be changed for all the respective instances?

Notwithstanding my typo of `%s` instead of `%2`, I don't follow what you mean. I'm proposing...
```suggestion
%2\setup-x86_64.exe --quiet-mode --root %CYGWIN_ROOT% --site http://cygwin.mirror.constant.com --packages curl,diff,diffutils,git,m4,make,patch,perl,rsync,mingw64-x86_64-gcc-core,unzip
```
as that's the only path in the script using forward slashes (and which will fail if `%2` is not an absolute path)

Reply:
Ah, changing to the use of  `%s` was exactly what I was expressing reluctance about...
I previously updated the `copy ...` path, and, just now, changed the original setup executable path as you now suggested.
When/if acceptable, I'll squash all the commits into one.